### ISSUE TITLE
Custom fields functional only after page is reloaded

### ISSUE THREAD

**Issue by ghost** [bitwarden/clients#7308]
### Steps To Reproduce

Please refer to 'Additional Context'

EDIT : I've just tested with previous versions of Bitwarden available on AMO:
Version 2023.12.0 : ISSUE
Version 2023.10.2 : ISSUE
Version 2023.10.1 : ISSUE
Version 2023.9.2 : NO ISSUE
Hence the culprit, within my environment, is tied to Bitwarden versions starting 2023.10.1

EDIT 2: I know for sure that there were no issues on Dec. 12, 2023
At the time Firefox and Bitwarden installed versions were :
- Mozilla Firefox 115.5.0 ESR : upgraded on Nov. 21, 2023
- Version 2023.12.0 Released/Updated Dec 6, 2023
So it seems that the culprit is in the pair Firefox 115.6.0 & Bitwarden versions starting 2023.10.1

### Screenshots or Videos

Initially:
![1](https://github.com/bitwarden/clients/assets/65161029/b4f88550-233f-488b-9336-dc0cda8cbe07)

Once page is reloaded:
![2](https://github.com/bitwarden/clients/assets/65161029/55d44887-411e-4a47-936d-c9e40df19ce3)

Additional Context

Hi,

I'm encountering an odd issue with custom fields which work perfectly only after the applicable page has been reloaded.
This is recent, previously, for years, never encountered this oddity.
I seldom use the very few custom fields I've set so I'm unable to determine if the culprit is within latest Bitwarden versions, or within Firefox (no issue in the past but until what version?), or within any browser tweak I may have set.

I just cannot figure out why the custom fields get correctly filled but only once the page has been reloaded.
Test url (but same issue and scenario on other urls) : https://www.ghacks.net/2023/12/21/you-may-now-hide-embarrassing-games-on-steam/
Custom fields names here are _author_ and _email_

Thanks for advice, hints, help :)

### Operating System

Windows

### Operating System Version

7

### Web Browser

Firefox

### Browser Version

115.6.0 ESR

### Build Version

2023.12.1

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by NovaSilentium** [comment]
Hi there!

We appreciate you letting us know of this issue. Due to the nature of autofill issues, we track them internally using different Forms depending on the platform:

mobile: https://docs.google.com/forms/d/e/1FAIpQLScMopHyN7KGJs8hW562VTzbIGL4KcFnx0wJcsW0GYE1BnPiGA/viewform

browser extension: https://docs.google.com/forms/d/e/1FAIpQLSfkxh1w6vK8fLYwAbAAEVhvhMAJwfFNDtYtPUVk1y5WTHvJmQ/viewform

In this case, we will go ahead and file an autofill report for you with the information you added here.

In the future, if you encounter any other page or app that doesn't autofill as expected, please avoid opening a new GitHub issue and go directly to the corresponding form.

This issue will now be closed. 

Thanks!

**Comment by Cade66** [comment]
> In the future, if you encounter any other page or app that doesn't autofill as expected, please avoid opening a new GitHub issue and go directly to the corresponding form.

There are domains, services we avoid, block, totally or partially, system-wide and/or browser-wide only. Should I encounter another Bitwrden autofill issue looks like I'll have to keep it for myself then : the forms you mention won't be accessed.

I've always considered that when an application has a GitHub repository it was there -- here therefor -- that any issues should be addressed. All nuances of what is offered by GitHub is not obvious for non techies but at least it's understandable. Having to consider from here another depository of issues is a path I didn't have in mind.

Thanks for having taken into consideration this thread of mine, and thanks to the Bitwarden team for their achievements.

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
